specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/OutboundFirewallRules.json

```diff
@@ -0,0 +1,284 @@
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "Outbound firewall rule does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Deletes a outbound firewall rule with a given name.": {
+            "$ref": "./examples/OutboundFirewallRuleDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/outboundFirewallRules": {
+      "get": {
+        "tags": [
+          "OutboundFirewallRules"
+        ],
+        "description": "Gets all outbound firewall rules on a server.",
+        "operationId": "OutboundFirewallRules_ListByServer",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved outbound firewall rules.",
+            "schema": {
+              "$ref": "#/definitions/OutboundFirewallRuleListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 OutboundFirewallRulesNotAllowedOnSynapse - Outbound Firewall Rules are not allowed on SQL DB servers connected to Synapse\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets list of outbound firewall rules on a server.": {
+            "$ref": "./examples/OutboundFirewallRuleList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OutboundFirewallRuleProperties": {
+      "description": "The properties of an outbound firewall rule.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The state of the outbound rule.",
+          "type": "string",
+          "readOnly": true
+        }
```

Comment:
note that properties.provisioningState cannot be any string, it needs to follow rpc.